src/menu.js

```diff
@@ -0,0 +1,79 @@
+import { gel, crel } from './utils';
+import {
+  salads,
+  shellfish,
+  fish,
+  meat,
+  wine,
+} from './menuItems';
+
+function renderHeader(content, text) {
+  const saladsHeader = crel('div');
+  saladsHeader.className = 'menuHeader';
+  saladsHeader.innerHTML = text;
+  content.appendChild(saladsHeader);
+}
+
+function renderItem(content, item) {
+  const menuItem = crel('div');
+  menuItem.className = 'menuItem';
+
+  const itemImage = crel('div');
+  itemImage.className = 'itemImage '.concat(item.image);
+  // itemImage.style.backgroundImage = 'url("../resources/images/'.concat(item.image).concat('")');
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
Thank you Oluwaseun for reviewing my request.  
I did all changes you suggested.